frontend/src/metabase/query_builder/components/NativeQueryEditor/VisibilityToggler/VisibilityToggler.jsx

```diff
@@ -29,7 +29,7 @@ const VisibilityToggler = ({
   );
 
   return (
-    <Container>
+    <Container className="hide sm-show">
```

Comment:
Could this migrated to emotion? e.g.
```
display: none;

${breakpointMinSmall} {
  display: block;
}
```

Reply:
Yeah, I'll update this.

Reply:
This actually isn't needed anymore, it was leftover from an earlier design idea. Removing